test/unit/app/get-featured-image.js

```diff
@@ -1,4 +1,4 @@
-import GetFeaturedImage from '../../src/app/lib/get-featured-image';
+import GetFeaturedImage from '../../../src/app/lib/get-featured-image';
```

Comment:
The convention is to capitalise the first letter if it is a constructor intended to be used with `new`. Never a function.